ports/mapnik/portfile.cmake

```
@@ -0,0 +1,111 @@
+    "input-postgis"             USE_PLUGIN_INPUT_POSTGIS
+    "input-raster"              USE_PLUGIN_INPUT_RASTER
+    "input-shape"               USE_PLUGIN_INPUT_SHAPE
+    "input-sqlite"              USE_PLUGIN_INPUT_SQLITE
+    "input-topojson"            USE_PLUGIN_INPUT_TOPOJSON
+    "viewer"                    BUILD_DEMO_VIEWER
+    "demo"                      BUILD_DEMO_CPP
+    "utility-geometry-to-wkb"   BUILD_UTILITY_GEOMETRY_TO_WKB
+    "utility-mapnik-index"      BUILD_UTILITY_MAPNIK_INDEX
+    "utility-mapnik-render"     BUILD_UTILITY_MAPNIK_RENDER
+    "utility-ogrindex"          BUILD_UTILITY_OGRINDEX
+    "utility-pgsql2sqlite"      BUILD_UTILITY_PGSQL2SQLITE
+    "utility-shapeindex"        BUILD_UTILITY_SHAPEINDEX
+    "utility-svg2png"           BUILD_UTILITY_SVG2PNG
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS   
+        ${FEATURE_OPTIONS}
+        -DCOPY_LIBRARIES_FOR_EXECUTABLES=OFF
+        -DCOPY_FONTS_AND_PLUGINS_FOR_EXECUTABLES=OFF
+        -DINSTALL_DEPENDENCIES=OFF
+        -DBUILD_TEST=OFF
+        -DBUILD_BENCHMARK=OFF
+        -DUSE_EXTERNAL_MAPBOX_GEOMETRY=ON
+        -DUSE_EXTERNAL_MAPBOX_POLYLABEL=ON
+        -DUSE_EXTERNAL_MAPBOX_PROTOZERO=ON
+        -DUSE_EXTERNAL_MAPBOX_VARIANT=ON
+        -DINSTALL_CMAKE_DIR=share/${PORT}/cmake
+        -DFONTS_INSTALL_DIR=share/${PORT}/fonts
+)
+
+vcpkg_cmake_install()
+
+# copy plugins into tool path, if any plugin is installed
+if(IS_DIRECTORY "${CURRENT_PACKAGES_DIR}/bin/plugins")
+  file(COPY "${CURRENT_PACKAGES_DIR}/bin/plugins" DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}")
+endif()
+vcpkg_copy_pdbs()
+
+if("demo" IN_LIST FEATURES)
+  file(COPY "${SOURCE_PATH}/demo/data" DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}/demo")
+  vcpkg_copy_tools(TOOL_NAMES mapnik-demo AUTO_CLEAN)
+endif()
+
+if("viewer" IN_LIST FEATURES)
+  # copy the ini file to reference the plugins correctly
+  file(COPY "${CURRENT_PACKAGES_DIR}/bin/viewer.ini" DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}")
+  vcpkg_copy_tools(TOOL_NAMES mapnik-viewer AUTO_CLEAN)
+endif()
+
+if("utility-geometry-to-wkb" IN_LIST FEATURES)
+  vcpkg_copy_tools(TOOL_NAMES geometry_to_wkb AUTO_CLEAN)
+endif()
+
+if("utility-mapnik-index" IN_LIST FEATURES)
+  vcpkg_copy_tools(TOOL_NAMES mapnik-index AUTO_CLEAN)
+endif()
+if("utility-mapnik-render" IN_LIST FEATURES)
+  vcpkg_copy_tools(TOOL_NAMES mapnik-render AUTO_CLEAN)
+endif()
+if("utility-ogrindex" IN_LIST FEATURES)
+  # build is currently not supported
+  # vcpkg_copy_tools(TOOL_NAMES ogrindex AUTO_CLEAN)
+endif()
+if("utility-pgsql2sqlite" IN_LIST FEATURES)
+  vcpkg_copy_tools(TOOL_NAMES pgsql2sqlite AUTO_CLEAN)
+endif()
+if("utility-shapeindex" IN_LIST FEATURES)
+  vcpkg_copy_tools(TOOL_NAMES shapeindex AUTO_CLEAN)
+endif()
+if("utility-svg2png" IN_LIST FEATURES)
+  vcpkg_copy_tools(TOOL_NAMES svg2png AUTO_CLEAN)
+endif()
+
+vcpkg_cmake_config_fixup(CONFIG_PATH "share/${PORT}/cmake")

```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH share/mapnik/cmake)
```
We suggest to use the port name directly instead of ${PORT} in this case. Also for relative path, it's not necessary to add quotes for it.